vector/src/main/res/drawable/ic_fab_add_by_qr_code.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file will also need to be replaced. Perhaps a QR code with a plus sign in the corner?